Bottom: dd0f19b489b15dbd505ad834e48add1490964c53
Top:    345b9581adf34f87452e92f0f220c152cca00c6d
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-31 17:56:38 +0800

Refresh of autofs-5.0.7-teach-dumpmaps-to-output-simple-key-value-pairs.patch

---

diff --git a/lib/master.c b/lib/master.c
index 3131a2e..3f2597b 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1356,7 +1356,7 @@ static int match_map_name(struct map_source *source, const char *name)
 		}
 	}
 
-	return;
+	return 0;
 }
 
 static int compare_source_type(struct map_source *map, const char *type)
@@ -1393,7 +1393,6 @@ int dump_map(struct master *master, const char *type, const char *name)
 		struct master_mapent *this;
 		struct autofs_point *ap;
 		time_t now = time(NULL);
-		int i;
 
 		this = list_entry(p, struct master_mapent, list);
 		p = p->next;
